### PR TITLE
Add th-TH.js

### PR DESCRIPTION
**Pull request by NatJNP**
I add Thai Language.

### CODE REVIEW

**Comment by SergioCrisostomo** [comment]
@NatJNP nice work! Thanks!

Can you please add 2 more lines?
Check https://github.com/iview/iview/blob/v2.7.3/src/locale/lang/en-US.js and look at line `#5` and line `#25`. Please add that also to the file.

**Comment by NatJNP** [comment]
I add line#5 and line#25 completion.

File: src/locale/lang/th-TH.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            clearFilter: 'ทั้งหมด'
+        },
+        datepicker: {
+            selectDate: 'เลือกวัน',
+            selectTime: 'เลือกเวลา',
+            startTime: 'เริ่มเวลา',
+            endTime: 'สิ้นสุดเวลา',
+            clear: 'ล้างข้อมูล',
+            ok: 'ตกลง',
+            datePanelLabel: '[ดดดด] [ปปปป]',
```
Comment (SergioCrisostomo) [inline src/locale/lang/th-TH.js]: @NatJNP sorry, maybe I was unclear. This line should follow the syntax here: https://github.com/iview/iview/blob/2.0/src/components/date-picker/util.js#L67-L74

You can see for example the [`zh-CN` file](https://github.com/iview/iview/blob/2.0/src/locale/lang/zh-CN.js#L25) to help.

**Comment by icarusion** [comment]
@NatJNP thanks for your pr, can you change your language package like this :https://github.com/iview/iview/blob/v2.7.3/src/locale/lang/en-US.js ?

**Comment by SergioCrisostomo** [comment]
@NatJNP we have added new features and now the files should have `setLang` for the i18n settings. Would be great if you add that. 👍

**Comment by NatJNP** [comment]
Okay I will do it.

**Comment by NatJNP** [comment]
I want to show thai language in  your website. How can i do it?

**Comment by SergioCrisostomo** [comment]
@NatJNP thank you! 
When we release next version we will update the website documentation to show that `th-TH.js` is available.

**Comment by NatJNP** [comment]
@SergioCrisostomo Okay Thank you.

**Pull request merged**